modules/sugar/src/test/ts/browser/FocusTest.ts

```diff
@@ -1,11 +1,16 @@
-import { assert, UnitTest } from '@ephox/bedrock-client';
+import { Assert, assert, UnitTest } from '@ephox/bedrock-client';
 import * as Compare from 'ephox/sugar/api/dom/Compare';
 import * as Focus from 'ephox/sugar/api/dom/Focus';
 import * as Insert from 'ephox/sugar/api/dom/Insert';
 import * as Remove from 'ephox/sugar/api/dom/Remove';
 import * as Body from 'ephox/sugar/api/node/Body';
 import Element from 'ephox/sugar/api/node/Element';
 import * as Attr from 'ephox/sugar/api/properties/Attr';
+import { withShadowElement } from 'ephox/sugar/test/WithHelpers';
+import { Option, OptionInstances } from '@ephox/katamari';
+import tOption = OptionInstances.tOption;
```

Comment:
This line looks wrong?

Reply:
I've used this a bunch of times. From the TS manual, it looks like it's supposed to be for `import x = require('y')`, but it works for this. 

It's probably a bad idea, so I'll remove it.

Reply:
Yeah, seems like a bug that even works. We also can't use `require` atm IIRC, so we definitely shouldn't be using that.

Reply:
Fixed.